doc/operation-and-maintenance/Logging-&-monitoring.md

```diff
@@ -84,60 +84,47 @@ By default, the resolution is set to 60 seconds.
-
-Wait a while, then open:
+```
+Session count:                   <prefix>.global.totalSessionCount.value
+XMPP messages received:          <prefix>.<domain>.xmppMessageReceived.one
+XMPP messages sent:              <prefix>.<domain>.xmppMessageSent.one
+Successful logins:               <prefix>.<domain>.sessionSuccessfulLogins.one
+Logouts:                         <prefix>.<domain>.sessionLogouts.one
+Authorization time:              <prefix>.<domain>.backends.auth.authorize.<value-type>
+RDBMS "simple" query time:       <prefix>.<domain>.backends.mongoose_rdbms.query.<value-type>
+RDBMS prepared query time:       <prefix>.<domain>.backends.mongoose_rdbms.execute.<value-type>
+MAM lookups:                     <prefix>.<domain>.mam_lookup_messages.one
+MAM archivization time:          <prefix>.<domain>.backends.mod_mam.archive.<value-type>
+MAM lookup time:                 <prefix>.<domain>.backends.mod_mam.lookup.<value-type>
+MAM private messages flush time: <prefix>.<domain>.mod_mam_odbc_async_pool_writer.flush_time.<value-type>
+MAM MUC messages flush time:     <prefix>.<domain>.mod_mam_muc_odbc_async_pool_writer.flush_time.<value-type>
+```
 
-    http://172.17.0.2/render?from=-10mins&until=now&target=stats.example
+Note that RDBMS metrics are only relevant if MongooseIM is [configured with an RDBMS backend](../advanced-configuration/database-backends-configuration.md), MAM metrics when [mod_mam is enabled](../modules/mod_mam.md) and MAM flush times when MAM is configured with an RDBMS backend with `async_writer` option (default).
 
-Then, if you configure your MongooseIM to send Exometer reports to that IP address and run it for a while, you should be able to see some interesting charts.
+#### Example graph in Grafana
 
-### Run graphite in Docker - alternative method
+![An example graph in Grafana](example-grafana-graph.png)
 
-If one of these steps doesn't work for you (e.g. they may be incompatible with your Docker on Mac), please try a less advanced alternative.
-It doesn't require a docker machine and will expose Graphite ports on localhost, instead of using a new route.
+This screenshot shows a graph plotting the RDBMS simple query time metric mentioned above.
+The graph is plotted for three nodes with each node having a different prefix: `mongoose.node1`, `mongoose.node2` and `mongoose.node3`.
 
-Start a container with Graphite:
+The queries take metrics for all nodes and all domains (`**` is a wildcard for multiple parts of the metric name) and groups them *per-node*, *per-value-type* - respectively `1`st (indexed from `0`) and `-1`st (last) parts of the metric's name.
 
-    $ docker run -d --name graphite --restart=always -p 80:80 -p 2003-2004:2003-2004 -p 2023-2024:2023-2024 -p 8125:8125/udp -p 8126:8126 hopsoft/graphite-statsd
+Because time-based metrics in MongooseIM are given in **microseconds**, to display understandable values in graph's legend the Y-axis unit has to be changed on the `Axes` tab.
```

Comment:
alt:
Time-based metrics in MongooseIM are given in **microseconds**, so to display human-readable values in graph's legend, the Y-axis unit has to be edited in the`Axes`tab.